### PR TITLE
[zh] docs/setup/production-environment/container-runtimes.md

### PR DESCRIPTION
**Pull request by wuyanpinguniontech**
fix docs/setup/production-environment/container-runtimes.md
Modify translated sentences

### CODE REVIEW

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ part of Kubernetes (this removal was
 [announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
 as part of the v1.20 release).
 -->
-v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 **dockershim** 的组件。 
+kubernetes v1.24 之前的版本，使用 **dockershim** 组件直接整合了 Docker Engine 。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:58]:
```suggestion
Kubernetes v1.24 之前的版本，使用 **dockershim** 组件直接整合了 Docker Engine 。
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -66,16 +66,16 @@ to understand how this removal might
 affect you. To learn about migrating from using dockershim, see
 [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).
 -->
-你可以阅读[检查 Dockershim 弃用是否会影响你](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/) 
+你可以阅读[检查弃用 Dockershim 是否会影响你](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:69]:
```suggestion
你可以阅读[检查移除 Dockershim 是否对你有影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 3]
```diff
@@ -172,8 +172,8 @@ resource pressure.
 单个 cgroup 管理器将简化分配资源的视图，并且默认情况下将对可用资源和使用
 中的资源具有更一致的视图。
 当有两个管理器共存于一个系统中时，最终将对这些资源产生两种视图。
-在此领域人们已经报告过一些案例，某些节点配置让 kubelet 和 docker 使用
-`cgroupfs`，而节点上运行的其余进程则使用 systemd; 这类节点在资源压力下
+在这方面，人们已经报告过一些示例，某些节点的 kubelet 和 docker 使用
+`cgroupfs`，而节点上运行的其余进程则使用 `systemd`，这类节点在资源压力下
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:176]:
```suggestion
`cgroupfs`，而节点上运行的其余进程则使用 `systemd`，
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 4]
```diff
@@ -172,8 +172,8 @@ resource pressure.
...
 会变得不稳定。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:177]:
```suggestion
这类节点在资源压力下会变得不稳定。
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 5]
```diff
@@ -298,10 +298,10 @@ using the (deprecated) v1alpha2 API instead.
 -->
 ## CRI 版本支持 {#cri-versions}
 
-你的容器运行时必须至少支持容器运行时接口的 v1alpha2。
+你的容器运行时必须至少支持v1alpha2 版本的容器运行时接口。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:301]:
```suggestion
你的容器运行时必须至少支持 v1alpha2 版本的容器运行时接口。
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 6]
```diff
@@ -298,10 +298,10 @@ using the (deprecated) v1alpha2 API instead.
...
 
-Kubernetes {{< skew currentVersion >}} 默认使用 v1 的 CRI API。如果容器运行时不支持 v1 API，
-则 kubelet 会回退到使用（已弃用的）v1alpha2 API。
+Kubernetes {{< skew currentVersion >}} 默认使用 v1 版本的 CRI API。如果容器运行时不支持 v1 版本的API，
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:303]:
```suggestion
Kubernetes {{< skew currentVersion >}} 默认使用 v1 版本的 CRI API。如果容器运行时不支持 v1 版本的 API，
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 7]
```diff
@@ -298,10 +298,10 @@ using the (deprecated) v1alpha2 API instead.
...
+则 kubelet 会回退到使用（已弃用的）v1alpha2 版本的API。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:304]:
```suggestion
则 kubelet 会回退到使用（已弃用的）v1alpha2 版本的 API。
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 8]
```diff
@@ -319,7 +319,7 @@ Use the following commands to install Containerd on your system:
 -->
 本节概述了使用 containerd 作为 CRI 运行时的必要步骤。
 
-使用以下命令在系统上安装 Containerd：
+使用以下命令在系统上安装 Containerd ：
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:322]:
```suggestion
使用以下命令在系统上安装 Containerd：
```
This space is not necessary. Same below.

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 9]
```diff
@@ -453,7 +453,7 @@ You should also note the changed `conmon_cgroup`, which has to be set to the val
 cgroup driver configuration of the kubelet (usually done via kubeadm) and CRI-O
 in sync.
 -->
-你还应该注意到 `conmon_cgroup` 被更改，当使用 CRI-O 和 `cgroupfs` 时，必须将其设置为值 `pod`。
+你还应该注意当使用CRI-O时，并且CRI-O 的 cgroup 设置为 `cgroupfs` 时，必须将`conmon_cgroup` 设置为值 `pod`。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:456]:
```suggestion
你还应该注意当使用 CRI-O 时，并且 CRI-O 的 cgroup 设置为 `cgroupfs` 时，必须将 `conmon_cgroup` 设置为值 `pod`。
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md (hunk 4 above)
Comment (wuyanpinguniontech) [reply content/zh-cn/docs/setup/production-environment/container-runtimes.md:177]: Thanks for your suggestions.

**Comment by windsonsea** [comment]
Could you squash your commits? Refer to [Squashing commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits).

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 10]
```diff
@@ -374,15 +374,15 @@ if you made changes to that file, also restart `containerd`.
 如果你从软件包（例如，RPM 或者 `.deb`）中安装 containerd，你可能会发现其中默认禁止了
 CRI 集成插件。
 
-你需要启用 CRI 支持才能在 Kubernetes 集群中使用 containerd。
+你需要启用 CRI 支持才能在 Kubernetes 集群中使用 containerd 。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:377]: The space is not necessary.

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 11]
```diff
@@ -374,15 +374,15 @@ if you made changes to that file, also restart `containerd`.
...
 要确保 `cri` 没有出现在 `/etc/containerd/config.toml` 文件中 `disabled_plugins`
 列表内。如果你更改了这个文件，也请记得要重启 `containerd`。
 {{< /note >}}
 
 <!--
 If you apply this change, make sure to restart containerd:
 -->
-如果你应用此更改，请确保重新启动 containerd：
+如果你应用此更改，请确保重新启动 containerd ：
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:385]: The space is not necessary.

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 12]
```diff
@@ -198,7 +198,7 @@ configuration, or reinstall it using automation.
 别的 cgroup 驱动，当为现有 Pods 重新创建 PodSandbox 时会产生错误。
 重启 kubelet 也可能无法解决此类问题。
 
-如果你有切实可行的自动化方案，使用其他已更新配置的节点来替换该节点，
+除非你有切实可行的自动化方案，使用其他已更新配置的节点来替换该节点，
```
Comment (tengqm) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:201]: 改错了

Comment (wuyanpinguniontech) [reply content/zh-cn/docs/setup/production-environment/container-runtimes.md]: Has been modified

**Comment by mahmut-Abi** [comment]
@tanjunchen

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md (hunk 10 above)
Comment (windsonsea) [reply content/zh-cn/docs/setup/production-environment/container-runtimes.md:377]: Not modified but click Resolved...

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md (hunk 11 above)
Comment (windsonsea) [reply content/zh-cn/docs/setup/production-environment/container-runtimes.md:385]: Not modified but click Resolved...

**Review by Sea-n — COMMENTED** [review]
/approve

**Comment by Sea-n** [comment]
Thanks for your contribution, this PR is ready for merge! 🎉

**Pull request merged**